packages/p2p/src/components/scrollbar-wrapper/scrollbar-wrapper.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useStore } from '@deriv/stores';
+
+type TScrollbarWrapper = {
+    children: JSX.Element;
+    height?: string;
+};
+
+const ScrollbarWrapper = ({ children, height }: TScrollbarWrapper) => {
+    const {
+        ui: { is_mobile },
+    } = useStore();
+
```

Comment:
or `const  { ui } = useStore(); const { is_mobile } = ui;` just curious.  🤷‍♀️

Reply:
you could do that, but I do see other components using this way as well 👀 honestly depends on preference, but since its so short, I just did it this way